.github/workflows/coverage-linux.yml

```diff
@@ -23,14 +21,22 @@ jobs:
           python-version: ${{ env.PYTHON_VERSION }}
       - name: Environment Information
         run: npx envinfo
+      - name: Clone gcovr reporter
+        run: git clone -b 3.4 --depth=1 --single-branch https://github.com/gcovr/gcovr.git
+      - name: Clone patch for gcovr
```

Comment:
@mhdawson do you remember why we're running gcovr from a branch and floating a patch? It would be nice to simplify this.

Reply:
It been long enough that I don't remember the specifics. I assume it failed to run properly with out that patch. It's possible a later version of gcovr may have fixed it.

Reply:
@mhdawson I've managed to get `gcovr@4.2` working, installed directly from `pip`.

The trick was to continue running from the `out/` folder, but to specify a `--root` pointing to the root project directory.